src/Game.ts

```diff
@@ -425,5 +425,30 @@ export abstract class Game {
     Audio.stopMusic(musicFile);
   }
 
+  /**
+   * Determines the game language files contain a entry for the specified GXT key
+   *
+   * @param entry - The GXT key.
+   * @returns true if GXT entry exists; otherwise, false
+   * @constructor
+   */
+  public static DoesGXTEntryExist(entry: number | string): boolean {
```

Comment:
Please use camelCase here and for all other methods throughout the PR :)

Reply:
Thanks, I'll check any code before merge :)